eliot/tests/test_prettyprint.py

```diff
@@ -0,0 +1,155 @@
+"""
+Tests for C{{eliot.prettyprint}}.
+"""
+
+from __future__ import unicode_literals
+
+from unittest import TestCase
+from subprocess import check_output, Popen, PIPE
+
+from six import PY3
+
+if PY3:
+    from .._py3json import dumps
+else:
+    from json import dumps
```

Comment:
Lots of copies of this pattern now.  Might be nice to have a centralized eliot json module that does this once and is used throughout the rest of the code.